reactor-core/src/test/java/reactor/core/publisher/MonoCallableTest.java

```diff
@@ -24,22 +24,20 @@
 import org.assertj.core.data.Offset;
 import org.junit.jupiter.api.Test;
 import org.reactivestreams.Subscription;
-
 import reactor.test.StepVerifier;
 import reactor.test.subscriber.AssertSubscriber;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
-import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class MonoCallableTest {
 
     @Test
     public void nullCallable() {
-		assertThrows(NullPointerException.class, () -> {
-			Mono.<Integer>fromCallable(null);
-		});
-	}
+        assertThatExceptionOfType(NullPointerException.class).isThrownBy(() -> {
```

Comment:
tabs is ideally what we'd use everywhere, so switching to tabs on modified lines was 👍